example/gluon/snail/README.md

```diff
@@ -0,0 +1,57 @@
+# SNAIL with Gluon
```

Comment:
This markdown requires a license in the form of comment. You may refer to other markdowns in the project for an example.

Reply:
add the license info https://github.com/apache/incubator-mxnet/pull/14367/commits/30f2ffad074e22d2f68aef5dc25ecb4829096a27